MANIFEST.in

```
@@ -8,15 +8,13 @@ recursive-include distributed *.svg
 recursive-include distributed *.ico
 recursive-include distributed *.yaml
 recursive-include docs *.rst
+recursive-exclude distributed **/tests/*
 
 include setup.py
 include README.rst
 include LICENSE.txt
 include MANIFEST.in
 include requirements.txt
```

There's a file called `pytest_resourceleaks.py`. Do we want to exclude that as well?

```suggestion
include requirements.txt
exclude distributed/pytest_resourceleaks.py
```